content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -124,8 +124,11 @@ saitama_savings_acc.deposit(500)
 saitama_savings_acc.withdraw(320)
 ```
 
+### super()
+In a child class, a parent class can be referred to with the use of the `super()` function. The super function returns a temporary object of the superclass (parent class) that allows access to all of its methods to its child class. There's no need to specify the parent class name to access its methods. This function can be used both in single and multiple inheritances.
```

Comment:
What do you think of this:- ' In a child class, a parent class can be referenced by using the *super()* function. This function returns a temporary instance/object of the superclass (parent class) that gives access to all its methods to its child class. It is not necessary to specify the parent class name.'